src/components/PayNow.svelte

```diff
@@ -67,7 +70,11 @@
 						>
 					</Alert.Root>
 				{/if}
-				<Dialog.Description>Pay {rocketProduct.Price} sats now with Lightning</Dialog.Description>
+				<Dialog.Description
+					>Pay {rocketProduct.Price === 1
+						? `${rocketProduct.Price} sat`
+						: `${rocketProduct.Price} sats`} now with Lightning</Dialog.Description
```

Comment:
Before creating the sat invoice, using the form `x`k sats may not be very precise, so I did not use `formatSats()` here.